src/components/NcUserStatusIcon/NcUserStatusIcon.vue

```diff
@@ -170,6 +197,11 @@ export default {
 
 <style lang="scss" scoped>
 .user-status-icon {
+	// Custom colors for the svg icons, to not rely on server variables
+	--color-icon-online: #2D7B41;
+	--color-icon-busy: #DB0606;
+	--color-icon-away: #C88800;
+	--color-icon-offline: #6B6B6B;
```

Comment:
Should be probably better scoped names like

```suggestion
	--user-status-color-online: #2D7B41;
	--user-status-color-busy: #DB0606;
	--user-status-color-away: #C88800;
	--user-status-color-offline: #6B6B6B;
```

Because `--color-xxxx` seems to be dangerous close to what server provides.

Reply:
As this only affects the library currently, lets merge and fix it in main first and backport?